examples/seismic/tutorials/13_making_synthetics.ipynb

```diff
@@ -0,0 +1,1504 @@
+      "text/plain": [
+       "<Figure size 432x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "# Take the center slice in the x direction\n",
+    "# Remember that in Devito, indexing convention is [x, y, z] (need to flip for correct imshow display)\n",
+    "plt.imshow(reshaped[50].T, cmap='viridis', origin='lower')\n",
+    "plt.colorbar()\n",
+    "plt.show()"
```

Comment:
What about changing this to be like one of the nice pyvista plots you have below? e.g. the one from this cell:
```
orth_slices = grid.slice_orthogonal(x=200, y=200, z=-500)

orth_slices.plot(cmap='seismic', clim=[-0.01, 0.01])
```

Reply:
The data reshaping is a little involved, and I figure doing a reshape to plot it isn't ideal when the intention is to qc the initial reshape